FastBank.Infrastructure/Repository/TransactionRepository.cs

```diff
@@ -9,7 +9,7 @@ public class TransactionRepository : ITransactionRepository
     {
         private readonly IRepository _repository;
 
-        public const int TransactionPerPage = 10;
+        public const int TRANSACTION_PER_PAGE = 3;
```

Comment:
TRANSACTIONS (Plural), not TRANSACTION_PER_PAGE. Should be TRANSACTIONS_PER_PAGE = 3